README.md

```diff
@@ -14,3 +14,11 @@ The best way is to...
 
 Thanks,
 *Michael*
+
+## Build
+
+With docker installed and user part of docker group simple run:
```

Comment:
Move before "Changes" section. Also since pulling a 4GB Docker image instead of using a standard LaTeX installation is probably not the default solution for the average student I'd say:

"Compile `robot_motion_planning_summary.tex` with your usual LaTeX build environment or use [blang's LaTeX Docker image](https://github.com/blang/latex-docker):"

Those who use Docker should know what they're doing or need to install it, so you can drop "With docker installed and user part of docker group simple run".

Reply:
fixed

I just added the docker script, 'cause I don't like polluting my system and LaTeX installs files all over the place on Linux.

Reply:
Sure, it's a good solution. I just wanted to avoid that it sounds like it's required.